### PR TITLE
Day08

### PR DESCRIPTION
**Pull request by ArthurSoftvision1**

### CODE REVIEW

File: Curs04/src/views/index.js [hunk 1]
```diff
@@ -1,8 +1,96 @@
...
-    const result = [];
-    result.push(LoginPage({}))
     const app = document.querySelector('#app');
+    // call LoginPage function and display LoginPage using innerHTML method
     app.innerHTML = LoginPage();
+
+    // get the form by his ID
+   const loginFormSection = document.getElementById("form-login")
+
+    const myObj = {}
```
Comment (s-timofte) [inline Curs04/src/views/index.js:10]: shouldn't this object be reset whenever you are opening a new evaluation page to make sure data from the previous evaluations does not persist into new ones?

File: Curs04/src/views/index.js [hunk 2]
```diff
@@ -1,8 +1,96 @@
...
+
+        // get evaluationButton by his ID
+        const evaluationButton = document.getElementById("evaluation-page")
+
+        // add "click" event listener
+        evaluationButton.addEventListener("click", goToEvaluationsPage)
+
+        const getSelectorAll = document.getElementsByTagName("select")
+        getSelectorAll.forEach(function(el) {
+           const nameSelector = el.name
```
Comment (s-timofte) [inline Curs04/src/views/index.js]: do we really need these constants?

File: Curs04/src/views/index.js [hunk 3]
```diff
@@ -1,8 +1,96 @@
...
+    }
+
+    // go to evaluation page function
+    const goToEvaluationsPage = function() {
+        // call EvaluationPage function and display EvaluationPage using innerHTML method
+        app.innerHTML = EvaluationsPage();
+        const newEvaluationButton = document.getElementById("new-evaluation-page")
+
+        // add click event listener on newEvaluationButton
+        newEvaluationButton.addEventListener("click", goToNewEvaluationPage)
```
Comment (s-timofte) [inline Curs04/src/views/index.js:85]: like how the names are representative to their contents/purpose

File: Curs04/src/views/index.js [hunk 4]
```diff
@@ -1,8 +1,96 @@
...
+        // get username input value by his ID
+        const usernameValue = document.getElementById("username-field").value
+
+        // get password input value by his ID
+        const passwordValue = document.getElementById("password-field").value
+
+        // set alert message if the inputs not meet the conditions
+        const displayAlert = function(message){
+            const verify = document.getElementById('alert') || false;
+            if(!verify){
```
Comment (s-timofte) [inline Curs04/src/views/index.js:102]: What happens if the element already exists and you need to show a different error?

File: Curs04/src/views/new-evaluation.js [hunk 5]
```diff
@@ -1,5 +1,328 @@
...
+            technicalLevelSection: ['1', '3', '3', '3']
+         }
+        
+         const navigationLinks = ['Evaluations', 'New Evaluation', 'Logout']
+         
+         const detailsForm = [
+             '<input class="candidate_input" type="text" name="candidate" placeholder="Candidate"/>',
+             '<input class="interviewer_input" type="text" name="interviewer" placeholder="Interviewer"/>',
+             '<input class="calendar_input" type="password" name="date" placeholder="4/22/2012"/>',
+             `<a href="evaluation.html"> <img style="width: 35px; margin-top: 18px;" src="img/2019-200.png"></a>`
```
Comment (s-timofte) [inline Curs04/src/views/new-evaluation.js]: the style should be in less files not inline

**Review by s-timofte — APPROVED** [review]

File: Curs04/src/views/new-evaluation.js (hunk 5 above)
Comment (ArthurSoftvision1) [reply Curs04/src/views/new-evaluation.js]: 👍

File: Curs04/src/views/index.js (hunk 4 above)
Comment (ArthurSoftvision1) [reply Curs04/src/views/index.js:102]: You can set a new message in this case, in the if condition.

File: Curs04/src/views/index.js (hunk 3 above)
Comment (ArthurSoftvision1) [reply Curs04/src/views/index.js:85]: newEvaluationButton is the variable which contains our ID element (the ID of the button)
After that I added a click event listener and I called the goToNewEvaluationpage function where the new-evaluation page will be rendered.

File: Curs04/src/views/index.js (hunk 2 above)
Comment (ArthurSoftvision1) [reply Curs04/src/views/index.js]: I use this constants to select all the selects and to select their name and value.

Probably exists other ways...

File: Curs04/src/views/index.js (hunk 1 above)
Comment (ArthurSoftvision1) [reply Curs04/src/views/index.js:10]: It should. I'm working on it. :)

File: Curs04/src/views/index.js (hunk 3 above)
Comment (s-timofte) [reply Curs04/src/views/index.js:85]: yes, it is intuitive and simple to understand.

File: Curs04/src/views/index.js (hunk 4 above)
Comment (s-timofte) [reply Curs04/src/views/index.js:102]: if the element exists the instructions in the if will not be executed, right?

Comment (ArthurSoftvision1) [reply Curs04/src/views/index.js:102]: yes

File: Curs04/src/views/index.js (hunk 3 above)
Comment (ArthurSoftvision1) [reply Curs04/src/views/index.js:85]: At the moment yes. 👍

File: Curs04/src/views/index.js (hunk 4 above)
Comment (s-timofte) [reply Curs04/src/views/index.js:102]: so if you want to add a new message while a message is showing already, what will happen?

Comment (ArthurSoftvision1) [reply Curs04/src/views/index.js:102]: If I add a new message won't work :)

Comment (s-timofte) [reply Curs04/src/views/index.js:102]: yep, it won't :)

File: Curs04/src/views/evaluation.less [hunk 6]
```diff
@@ -0,0 +1,266 @@
...
+    margin: 8px 0;
+    color: white;
+    height: 100%;
+    font-size: 20px;
+    border-radius: 100%;
+    border: 1px solid @lightBlue;
+}
+
+/* Large screens ----------- */
+@media screen and (max-width: 1824px) and (min-width: 1200px) {
```
Comment (s-timofte) [inline Curs04/src/views/evaluation.less]: for readability it is usually `min-width` first si `max-width` last

File: Curs04/src/views/index.html [hunk 7]
```diff
@@ -4,18 +4,24 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
+
+    <link rel="stylesheet" href="img/">
+
+    <link rel="stylesheet/less" href="index.less" type="text/css"/>
```
Comment (s-timofte) [inline Curs04/src/views/index.html]: there should be only one less file imported... use the less @import instruction

http://lesscss.org/features/#features-overview-feature-importing#features-overview-feature-importing

**Review by s-timofte — APPROVED** [review]
Add new lines at the end of each file please. There should always be a new line at the end of any file you write code in,

File: Curs04/src/views/evaluation.less (hunk 6 above)
Comment (ArthurSoftvision1) [reply Curs04/src/views/evaluation.less]: 👍

File: Curs04/src/views/index.html (hunk 7 above)
Comment (ArthurSoftvision1) [reply Curs04/src/views/index.html]: Where should I put the @import instruction?

In html file won't work..

**Review by s-timofte — APPROVED** [review]

File: Curs04/src/views/index.html (hunk 7 above)
Comment (s-timofte) [reply Curs04/src/views/index.html]: in the main less file, check the link I left on the previous comment... those are all less instructions

**Pull request merged**